ivy/functional/backends/jax/linear_algebra.py

```diff
@@ -361,7 +361,7 @@ def svd(
     x: JaxArray, /, *, compute_uv: bool = True, full_matrices: bool = True
 ) -> Union[JaxArray, Tuple[JaxArray, ...]]:
     if compute_uv:
-        results = namedtuple("svd", "U S Vh")
+        results = namedtuple("svd", ['U', 'S', 'Vh'])
```

Comment:
I think changing this from a string to a list is unnecessary, right? Can you change this back for all the functions you've changed it in, unless there's a reason?

Reply:
sure, will do!